### ISSUE TITLE
Chrome OS boot to a black screen

### ISSUE THREAD

**Issue by MeguMario** [imperador/chromefy#93]
So, I made my image of Chrome OS using Option 1, booted it with my Asus X550UA, it works fine, then I switch to the X401A, only to be greeted with a big black screen, Ctrl+Alt+F2 doesn't do anything, Fn hotkeys works (Brightness & Turn off Screen)

**Comment by arminmacx** [comment]
Yes I have the same issue after finally successfully install chromeos now when it boot it shows google logo and then it goes to black screen and I just can see my mouse arrow which I can play with it. I can use ctrl + alt + f2 to go to console.

**Comment by arminmacx** [comment]
Ok I find it why I can’t see anything because of I use nvidia 640 gpu and looks like special build of atb and chromefy can’t build gpu driver for that

**Comment by dragon788** [comment]
I had similar weirdness until I found a comment that led me to the steps below.

Try making the modifications in [this comment](https://github.com/imperador/chromefy/issues/10#issuecomment-552267462) to the `write_gpt.sh` script to give the root partitions more space. You will need to do that after you are booted into a Chromefy USB, so if you aren't getting that far, try using the ArnoldTheBat r76 and a matching ChromeOS recovery at r76 to build your base image.

**Comment by Relz** [comment]
I made my image of Chrome OS using Option 1, booted it with my Lenovo G580 => Black screen with console cursor at top-left of screen.
Ctrl+Alt+F2 doesn't do anything.
Fn+F2 (Turn off screen) works.
FydeOS v8.21 works.
CloudReady: Home Edition v78.4 works.

**Comment by joyater** [comment]
so whats the fix?

**Comment by oxesoft** [comment]
I have the exact same issue with my Intel NUC8i3BEK.

**Comment by joyater** [comment]
same
